lib/maplibre/styles.ex

```
@@ -0,0 +1,4123 @@
+        "type" => "symbol"
+      },
+      %{
+        "filter" => ["all", ["==", "$type", "Point"], ["in", "class", "country"]],
+        "id" => "country_label",
+        "layout" => %{
+          "text-field" => "{name:latin}",
+          "text-font" => ["Noto Sans Bold"],
+          "text-max-width" => 10,
+          "text-size" => [
+            "interpolate",
+            ["linear", 0.75, 1, 0.75, 1],
+            ["zoom"],
+            1,
+            ["step", ["get", "rank"], 13, 1, 12, 2, 12],
+            4,
+            ["step", ["get", "rank"], 15, 1, 14, 2, 14],
+            6,
+            ["step", ["get", "rank"], 23, 1, 18, 2, 18],
+            9,
+            ["step", ["get", "rank"], 27, 1, 22, 2, 22]
+          ]
+        },
+        "maxzoom" => 12,
+        "paint" => %{
+          "text-color" => "rgba(255, 255, 255, 1)",
+          "text-halo-blur" => 1,
+          "text-halo-color" => "rgba(0, 0, 0, 1)",
+          "text-halo-width" => 1
+        },
+        "source" => "openmaptiles",
+        "source-layer" => "place",
+        "type" => "symbol"
+      }
+    ],
+    "metadata" => %{
+      "maptiler:copyright" =>
+        "This style was generated on MapTiler Cloud. Usage outside of MapTiler Cloud requires valid MapTiler Data Package: https://www.maptiler.com/data/package/ -- please contact us."
+    },
+    "name" => "Satellite Hybrid",
+    "pitch" => 0,
+    "sources" => %{
+      "openmaptiles" => %{
+        "type" => "vector",
+        "url" =>
+          "https://api.maptiler.com/tiles/v3/tiles.json?key=get_your_own_OpIi9ZULNHzrESv6T2vL"
+      },
+      "satellite" => %{
+        "tileSize" => 512,
+        "type" => "raster",
+        "url" =>
+          "https://api.maptiler.com/tiles/satellite-v2/tiles.json?key=get_your_own_OpIi9ZULNHzrESv6T2vL"
+      }
+    },
+    "version" => 8,
+    "zoom" => 1
+  }
+
+  def style(:street), do: @street
+  def style(:terrain), do: @terrain
+  def style(_), do: @default
```

Maybe it is better to raise an exception that says: "unknown style: ..."?